packages/GA4Client/test/src/tests.js

```diff
@@ -1497,14 +1512,10 @@ describe('Google Analytics 4 Event', function () {
             },
         };
 
-        beforeEach(function () {
-            window.dataLayer = [];
-        });
-
         it('should handle onUserIdentified with customerid', function (done) {
             kitSettings.externalUserIdentityType = 'CustomerId';
             mParticle.forwarder.init(kitSettings, reportService.cb, true, null);
-
+            dataLayer = [];
```

Comment:
Should this be `window.dataLayer` to be consistent with the rest of the usage?

Reply:
yes.  good catch